build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 set -e
-set -x
+#set -x
```

Comment:
@heplesser I documented the 'out-commenting'. When debugging the script, the flag is very [informative](http://julio.meroh.net/2010/01/set-e-and-set-x.html).